components/camel-jgroups-raft/src/main/docs/jgroups-raft-component.adoc

```diff
@@ -0,0 +1,195 @@
+[[jgroups-raft-component]]
+== JGroups raft Component
+*Available as of Camel version 2.24*
```

Comment:
multiple "available as of camel...." doc lines.

Reply:
Yeah better to remove the line and regen